README.md

```diff
@@ -2,6 +2,25 @@
 
 Cleisthenes is PBFT consensus library adaptable to any application.
 
+## Architecture
+
+### Logical View
+
+<p align="center">
+	<img src="./img/cleisthenes-architecture-logical-view.png" ></img>
+</p>
+
+**PBFT** basically communicate with **Application**. Whenever Application needs consensus with its network members, **Application** sends request to **PBFT**.
+
+If one of members in the network need to gossiping message to the network such as notifing changed leader or broadcasting byzantine node, we can gossiping those messages through **Messaging**. 
+
+**Messaging** gossips member's meta data such as whether this node is leader or not, participate in consensus or not and so on. And everytime node found that one of member changed, update states of **Members** 
+
+**Members** manage all the nodes information participate in the network. Whether node is alive or not, whether it is leader node or whether it is participate in consensus.
```

Comment:
According to my suggestion, `all the nodes` should be change to `all the members`.